seacrowd/sea_datasets/yunshan_cup_2020/yunshan_cup_2020.py

```diff
@@ -0,0 +1,164 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@article{DBLP:journals/corr/abs-2204-02658,
+ author    = {Yingwen Fu and
+              Jinyi Chen and
+              Nankai Lin and
+              Xixuan Huang and
+              Xin Ying Qiu and
+              Shengyi Jiang},
+ title     = {Yunshan Cup 2020: Overview of the Part-of-Speech Tagging Task for
+              Low-resourced Languages},
+ journal   = {CoRR},
+ volume    = {abs/2204.02658},
+ year      = {2022},
+ url       = {https://doi.org/10.48550/arXiv.2204.02658},
+ doi       = {10.48550/arXiv.2204.02658},
+ biburl    = {https://dblp.org/rec/journals/corr/abs-2204-02658.bib},
```

Comment:
Slightly modify the `_CITATION` to match their source repo
```suggestion
 doi       = {10.48550/arXiv.2204.02658},
 eprinttype = {arXiv},
 eprint    = {2204.02658},
 timestamp = {Tue, 12 Apr 2022 18:42:14 +0200},
 biburl    = {https://dblp.org/rec/journals/corr/abs-2204-02658.bib},
```